### PR TITLE
Dpl 919 transfer pbmc pools to 10x chip

### PR DESCRIPTION
**Pull request by yoldas**
Closes #

#### Changes proposed in this pull request

- Add the asset shape Shape8x1 for Chromium GEM-X chip to record loading.
- Rename purposes, request_types, and submission_templates to GEM-X versions

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

**Comment by yoldas** [comment]
Validation of submission templates and request_types name changes in local development database after executing post_deploy, without resetting the database:

- Limber-Htp - scRNA Core Chromium GEM-X 5p Aggregation <-> limber_scrna_core_chromium_gem_x_5p_aggregation
- Limber-Htp - scRNA Core Chromium GEM-X 5p <-> limber_scrna_core_chromium_gem_x_5p
- Limber-Htp - scRNA Core cDNA Prep Input <-> limber_scrna_core_cdna_prep_input
- Limber-Htp - scRNA Core cDNA Prep <-> limber_scrna_core_cdna_prep_v2 

NP. We added _v2 suffix (limber_scrna_core_cdna_prep_v2) in a previous modification of the submission and request types and it was a major change (we removed the definition that had multiple request types with one submission). We are not changing the request type name this time. 

Also, the library type has changed to Chromium single cell 5 prime GEM-X v2 and there is a record in the library_types table. Renaming from HT to GEM-X makes the library type name to retain the prefixes for library types. We will ask if this is OK.

File: config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.wip.yml [hunk 1]
```diff
@@ -22,15 +22,15 @@ Limber-Htp - scRNA Core cDNA Prep Input:
...
-    request_type_keys: ["limber_scrna_core_chromium_ht_5p_aggregation"]
+    request_type_keys: ["limber_scrna_core_chromium_gem_x_5p_aggregation"]
     product_line_name: Short Read
     product_catalogue_name: scRNA Core
-Limber-Htp - scRNA Core Chromium HT:
+Limber-Htp - scRNA Core Chromium GEM-X 5p:
   submission_class_name: "LinearSubmission"
   related_records:
-    request_type_keys: ["limber_scrna_core_chromium_ht"]
+    request_type_keys: ["limber_scrna_core_chromium_gem_x_5p"]
```
Comment (KatyTaylor) [inline config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.wip.yml:34]: Is the addition of '5p' here intentional?

**Review by KatyTaylor — COMMENTED** [review]
Just a couple of questions.

File: config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml [hunk 2]
```diff
@@ -16,10 +16,10 @@ limber_scrna_core_cdna_prep_v2:
     - LRC Bank Seq
     - LRC Bank Spare
   library_types:
-    - Chromium single cell 5 prime HT v2
-    # - Chromium single cell 3 prime HT v3
-    # - Chromium single cell BCR HT
-    # - Chromium single cell TCR HT
+    - Chromium single cell 5 prime GEM-X v2
+    # - Chromium single cell 3 prime GEM-X v3
```
[the remark this answers is not in the document]
Comment (yoldas) [reply config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml:20]: Actually we need to ask about these library type names but you are right. Presumably they should not have versioning suffixes.

[the remark this answers is not in the document]
Comment (yoldas) [reply config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml]: I have now removed the version suffixes from the library type names with GEM-X in Limber, SS, and Int. Suite PRs.

File: config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.wip.yml (hunk 1 above)
Comment (yoldas) [reply config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.wip.yml:34]: I think using 5p is consistent in this case.

**Pull request merged**